src/routes/safe/components/Transactions/TxList/TxSummary.tsx

```diff
@@ -10,21 +10,77 @@ import {
   isMultiSendTxInfo,
   isMultiSigExecutionDetails,
 } from 'src/logic/safe/store/models/types/gateway.d'
-import { InlineEthHashInfo } from './styled'
+import { InlineEthHashInfo, StyledGridRow } from './styled'
 import { NOT_AVAILABLE } from './utils'
 import TxShareButton from './TxShareButton'
 import { IS_PRODUCTION } from 'src/utils/constants'
 import TxInfoMultiSend from './TxInfoMultiSend'
 import DelegateCallWarning from './DelegateCallWarning'
+import styled from 'styled-components'
+import { getByteLength } from 'src/components/DecodeTxs' // must pass to utils
+import { generateSignaturesFromTxConfirmations } from 'src/logic/safe/safeTxSigner'
+import { makeConfirmation } from 'src/logic/safe/store/models/confirmation'
 
 type Props = { txDetails: ExpandedTxDetails }
 
+const FlexWrapper = styled.div<{ margin: number }>`
+  display: flex;
+  align-items: center;
+
+  > :nth-child(2) {
+    margin-left: ${({ margin }) => margin}px;
+  }
+`
+
+type TxDataRowType = { inlineType?: 'hash' | 'rawData'; title: string; value: string }
+
+const TxDataRow = ({ inlineType, title, value }: TxDataRowType) => (
+  <StyledGridRow>
+    <Text size="xl" as="span">
+      {title}
+    </Text>
+    {inlineType === 'hash' && (
+      <InlineEthHashInfo textSize="xl" hash={value} shortenHash={8} showCopyBtn explorerUrl={getExplorerInfo(value)} />
+    )}
+    {inlineType === 'rawData' && (
+      <FlexWrapper margin={5}>
+        <Text size="xl">{value ? getByteLength(value) : 0} bytes</Text>
+        <CopyToClipboardBtn textToCopy={value || ''} />
+      </FlexWrapper>
+    )}
+    {!inlineType && (
+      <Text size="xl" as="span">
+        {value}
+      </Text>
+    )}
+  </StyledGridRow>
+)
+
 export const TxSummary = ({ txDetails }: Props): ReactElement => {
   const { txHash, detailedExecutionInfo, executedAt, txData, txInfo } = txDetails
-  const explorerUrl = txHash ? getExplorerInfo(txHash) : undefined
-  const nonce = isMultiSigExecutionDetails(detailedExecutionInfo) ? detailedExecutionInfo.nonce : undefined
-  const created = isMultiSigExecutionDetails(detailedExecutionInfo) ? detailedExecutionInfo.submittedAt : undefined
-  const safeTxHash = isMultiSigExecutionDetails(detailedExecutionInfo) ? detailedExecutionInfo.safeTxHash : undefined
+
+  let created, confirmations, safeTxHash, baseGas, gasPrice, gasToken, refundReceiver, safeTxGas
+  if (isMultiSigExecutionDetails(detailedExecutionInfo)) {
+    // prettier-ignore
```

Comment:
Don't ignore, be it with Prettier or TypeScript.

Reply:
It kept adding a `;` before the parenthesis, breaking the destructuring... Can we leave this one?

Reply:
It's defining a new line with the `;`. If it's that much of a pain, leave it.